src/pages/LandingPage.js

```diff
@@ -1070,63 +1069,70 @@ class LandingPage extends React.Component {
           }
           landing_page_state={this.state.landing_page_state}
         />
-        <LandingPagePrompt
-          handlerSetLandingPageState={this.handlerSetLandingPageState}
-          landing_page_animations_middleTitle={
-            this.state.landing_page_animations_middleTitle
-          }
-          landing_page_state={this.state.landing_page_state}
-        />
-        <div
-          id="main-screen"
-          className={`desktop ${this.state.landing_page_state}`}
-        >
+        { this.state.landing_page_state ==
+          CONSTANTS.LANDING_PAGE_STATES.DESKTOP_LANDING_PAGE_LOAD &&
+          <LandingPagePrompt
+            handlerSetLandingPageState={this.handlerSetLandingPageState}
+            landing_page_animations_middleTitle={
+              this.state.landing_page_animations_middleTitle
+            }
+            landing_page_state={this.state.landing_page_state}
+          />
+        }
+        {this.state.landing_page_state ==
+          CONSTANTS.LANDING_PAGE_STATES.DEFAULT && (
```

Comment:
I know your intention with this is that you want the object to disappear after it does its transition, but what happens here is you end up killing it abruptly. 

I haven't figured out a good way to resolve this, but I think CSSTransition is a good way to do this: http://reactcommunity.org/react-transition-group/css-transition/

Only issue is I've been kinda just using a bunch of CSS rules to do this so it's hard to convert to using CSSTransition :/